src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -682,7 +682,7 @@ protected override void ProcessRecord()
                         {
                             if (exception.HResult == 299)
                             {
-                                WriteObject(PsUtils.GetMainModule(process).FileVersionInfo, true);
+                                WriteObject(process.MainModule.FileVersionInfo, true);
```

Comment:
I agree that it looks like a bug, but I didn't touch it because otherwise I would have changed existing behavior.
If you are OK to fix it in this PR, then it's fine for me.

Reply:
I believe we never want NRE in the context.

Reply:
Sure, my doubt was about doing that change in another PR rather than in this one.
I agree that we should do it while we're at it.